src/components/ListItemTexts/ListItemTextBase/index.tsx

```diff
@@ -0,0 +1,17 @@
+import {
+    ListItemText as MaterialListItemText,
+    ListItemTextProps as MaterialListItemTextProps,
+    Box as MaterialBox,
+} from "@material-ui/core";
+
+export interface ListItemTextBaseProps extends MaterialListItemTextProps {}
+
+const ListItemTextBase = (props: ListItemTextBaseProps) => {
+    return (
+        <MaterialBox color="primary.contrastText">

```

Comment:
I would prefer you use useTheme and write theme.palette.primary.contrastText instead. 
This is error-prone